File: LeetCode/Medium/7-reverse-integer/try-3.c
/*
 * Problem #7: Reverse Integer
 * Difficulty: Medium
 * Submission: Try 3
 * status: Accepted
 * Language: c
 * Date: 7/10/2026, 11:17:04 PM
 * Link: https://leetcode.com/problems/reverse-integer/
 */

int reverse(int x){
    long long a=x;
    long long rever=0;
    while(a!=0){
        rever=(10*rever)+(a%10);
        a/=10;
    }
    if(rever>=INT_MIN && rever<INT_MAX){
    return rever;
}
else
{
    return 0;
}

}

File: LeetCode/Easy/367-valid-perfect-square/try-3.py
/*
 * Problem #367: Valid Perfect Square
 * Difficulty: Easy
 * Submission: Try 3
 * status: Accepted
 * Language: python3
 * Date: 7/10/2026, 11:14:12 AM
 * Link: https://leetcode.com/problems/valid-perfect-square/
 */

class Solution:
    def isPerfectSquare(self, num: int) -> bool:
        if (math.sqrt(num)).is_integer():
            return True
        return False
        
